GitCommands/Git/GitSshHelpers.cs

```diff
@@ -26,19 +26,16 @@ bool DoArgumentsRequireSsh()
             }
         }
 
-        /// <summary>Un-sets the git SSH command path.</summary>
-        public static void UnsetSsh()
-        {
-            Environment.SetEnvironmentVariable("GIT_SSH", "", EnvironmentVariableTarget.Process);
-        }
-
         /// <summary>Sets the git SSH command path.</summary>
         public static void SetSsh(string? path)
         {
-            if (!Strings.IsNullOrEmpty(path))
+            if (Strings.IsNullOrEmpty(path))
             {
-                Environment.SetEnvironmentVariable("GIT_SSH", path, EnvironmentVariableTarget.Process);
+                // GIT_SSH is not required by Git to use OpenSSH, but supposedly by Gerrit plugin
+                path = _sshPathLocatorInstance.GetSshFromGitDir(AppSettings.GitBinDir) ?? "";
```

Comment:
🤔 If this is required by Gerrit, should this be in Gerrit plugin?

Reply:
Maybe
It does not seem like Gerrit uses GIT_SSH directly but relies on SshPathLocator.Find().
It could probably be OK to unset the variable if empty instead.
Pushing an update.

Gerrit maybe needs to implement SshPathLocator.GetSshFromGitDir() too, not sure how an empty path is handled. (There is code, not sure if that is working.)
The plugin imports GitUI so it need to be rebuilt on most GE changes anyway.